processor/filterprocessor/config.go

```diff
@@ -39,8 +48,30 @@ type Config struct {
 	Logs LogFilters `mapstructure:"logs"`
 
 	Spans SpanFilters `mapstructure:"spans"`
+
+	Conditions Conditions `mapstructure:"conditions"`
+}
+
+type Conditions struct {
```

Comment:
Somewhere there is a need to have a "list" of conditions, since I want to be able to remove spans based on more conditions.

Reply:
In OTTL this means AND or OR statements all within the same condition statement

Reply:
The conditions may get super complicated (for OR case especially, if I want to remove multiple metrics), I would prefer to have multiple simpler conditions, and if any of them matches then the telemetry is removed.

Reply:
Essentially if I have 3 complete independent conditions this force me to do a complicated 1 condition:
* Filter "healthcheck" spans
* Filter spans from service "envelope"
* Filter spans from pod "foo"

Will get super complicated.

Reply:
Ok, added multiple condition support.